actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -109,7 +109,7 @@ pub fn codecopy(state: &mut ExecutionState, code: &[u8]) -> Result<(), StatusCod
 
 pub fn call<BS: Blockstore, RT: Runtime<BS>>(
     state: &mut ExecutionState,
-    platform: &mut System<BS, RT>,
+    system: &mut System<BS, RT>,
```

Comment:
I think it'd be clearer to understand functional changes if we split renaming to a separate commit.